generators/app/templates/services/github/index.js

```diff
@@ -0,0 +1,28 @@
+import request from 'request-promise'
+import bluebird from 'bluebird'
+
+var userRequest = request({
+    uri: 'https://api.github.com/user',
+    json: true,
+    qs: {
+        access_token: accessToken
```

Comment:
It will not work because `accessToken` isn't defined. It's defined on the `getMe` scope only.
A solution would be transform `userRequest` in a function like `const userRequest = (accessToken) => request(...` and pass the accessToken from `getMe` (e.g. `bluebird.all([userRequest(accessToken)...`).